src/main/java/plugily/projects/villagedefense/handlers/HolidayManager.java

```diff
@@ -133,7 +131,7 @@ public HolidayType getCurrentHoliday() {
    *
    * @param zombie entity to apply effects for
    */
-  public void applyHolidayZombieEffects(Zombie zombie) {
+  public void applyHolidayZombieEffects(Creature zombie) {
```

Comment:
It is not zombie only anymore (rename) [Furthermore should be checked if its possible to sethelmet to all creatures]